README.md

```diff
@@ -75,62 +84,60 @@ flipr.init({
       input: 'user.userId',
     }
   ]
-})
+});
 
-var input = {
+const input = {
   user: {
     userId: '293890'
   }
 };
-flipr(input, function(err, config){
-  console.dir(config);
-});
-flipr(input, 'someConfigKey', function(err, value){
+
+async function complexExample() {
+  const config = await flipr.getConfig(input);
+  console.log(config);
+  const value = await flipr.getValue('someConfigKey', input);
   console.log(value);
-});
+}
+
+complexExample().catch();
 ```
 
 ## What is a flipr source?
-Good question.  A flipr source is something that gives flipr configuration data using a specific format, over a specific interface.  The format can be seen in the examples above.  Where does it get the data?  It's up to you.  There are some pre-built sources for flipr (see below), but you can easily create your own.  A flipr source should expose the following interface:
+A flipr source is something that gives flipr configuration data using a specific schema, over a specific interface.  The format can be seen in the examples above.  Where does it get the data?  It's up to you.  There are some pre-built sources for flipr (see below), but you can easily create your own.  A flipr source should expose the following interface:
```

Comment:
Are all of the exposed interfaces required now?

Reply:
Yep